test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
```

Comment:
I was thinking last night that it may be a good idea to have tests that trigger the event from a variety of methodologies, to ensure it's working

e.g.,

`domEl.click()` on the DOM element itself
`triggerEvent`
`$(domEl).trigger('click')` or `$(domEl).click()` 

If it fails with jQ trigger, it seems to imply then that it's not the triggering but the handler for the event